test/Frontend/emit-module-api-descriptor.swift

```diff
@@ -0,0 +1,10 @@
+// RUN: %empty-directory(%t)
+// RUN: echo 'print("Hello, World!")' >%t/main.swift
+// RUN: cd %t
+
+// RUN: %target-swift-frontend -emit-module -emit-api-descriptor-path %t/main.api.json %t/main.swift
+// RUN: test -f %t/main.api.json
+
+// RUN: echo '"%/t/main.swift": { api-descriptor-json: "%/t/foo.api.json" }' > %/t/filemap.yaml
```

Comment:
Are the `%/t`s intentional here or should they be `%t`?

Reply:
Never mind, I see this pattern in other tests; I'm just unfamiliar with what it's for.

Reply:
I admit I copied and pasted this from test/Frontend/emit-module-abi-descriptor.swift and did not notice this at first -- that seems a way to normalize file paths according to the [lit guide](test/Frontend/emit-module-api-descriptor.swift)

> 

%/s | %s but \ is replaced by /
-- | --




Macro | Substitution
-- | --
%/s | %s but \ is replaced by /